src/BaseTemplates/EmptyWeb/EmptyWeb.csproj

```diff
@@ -9,8 +8,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="**\*.cs" />
-    <EmbeddedResource Include="**\*.resx" />
+    <Compile Include="**\*.cs" Exclude="$(GlobalExclude)" />
```

Comment:
Just checking, the re-addition of all the Excludes of GlobalExclude is intentional, correct?